api/client/samples/similar_regions/similar_region.py

```diff
@@ -72,6 +72,8 @@ def _similar_to(self, region_id, number_of_regions):
         # list as an index to preserve dimensionality of returned data
         x = self.state.data_standardized[self.state.mapping[region_id], :]
         x = x.reshape(1, -1)
+        assert number_of_regions < self.ball.data.shape[0], "num_regions must be smaller than or equal to total " \
```

Comment:
Use the real variable name in the error message (i.e. num_regions or number_of_regions). And let's add the param to the function's docstring.

Reply:
the command line param is called num_regions so i'll change this to make it consistent, for error message, variable name and command line param.